collections/_posts/2022-06-28-MomentumResidualNeuralNetwork.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: review
+title: "Momentum Residual Neural Networks"
+tags: Invertible ResNets
+author: "Stéphanie Jehan-Besson"
+cite:
+    authors: "Michael E. Sander, Pierre Ablin, Mathieu Blondel, Gabriel Peyré"
+    title:   "Momentum Residual Neural Networks"
+    venue:   "ArXiv, publiseh in the 38th International Conference on Machine Learning, Aug 2021, Online, United States. pp.9276-9287."
```

Comment:
publiseh => published

Reply:
I think for brevity's sake, you could also simply indicate "ICML 2021". There's no need to mention that the paper is available on arXiv since you provide the link to it there on the very next line :wink: